python/lsst/cp/pipe/ptc.py

```diff
@@ -1072,14 +990,12 @@ def errFunc(p, x, y):
                                f" {Counter(mask)[False]} out of {len(meanVecOriginal)}"))
 
             if (len(meanVecFinal) < len(parsIniPtc)):
-                msg = (f"\nSERIOUS: Not enough data points ({len(meanVecFinal)}) compared to the number of"
+                msg = (f"\nSERIOUS: Not enough data points ({len(meanVecFinal)}) compared to the number of "
```

Comment:
Why do you start with a new line, is it to grab attention? Warn should start on a new line each time.

Reply:
I'm not sure why that was originally there (I did not add this part), but I deleted the `\n` given that you say that `warn` should already start at a new line.